docs/export_tf_graphs_guide.md

```diff
@@ -0,0 +1,74 @@
+# Exporting TensorFlow Graphs Guide
+
+This document describes how to export frozen TensorFlow graphs for inference.
+
+
+## Exporting pre-trained models from the TF-Slim Model Zoo
```

Comment:
As pointed out by @iantimmis, this can be done much more succinctly via our existing codebase: https://github.com/voxel51/theta/blob/develop/theta/classifiers/tfslim_classifiers.py#L1166-L1179

I will update